fe/fe-core/src/main/java/org/apache/doris/statistics/AnalysisTaskExecutor.java

```diff
@@ -50,19 +50,12 @@ public AnalysisTaskExecutor(int simultaneouslyRunningTaskNum) {
                     TimeUnit.DAYS, new LinkedBlockingQueue<>(),
                     new BlockedPolicy("Analysis Job Executor", Integer.MAX_VALUE),
                     "Analysis Job Executor", true);
+            cancelExpiredTask();
```

Comment:
This cancelExpiredTask will only executed once?

Reply:
It would submit a thread to do this job

Reply:
I see, cancelExpiredTask this function will create a thread.
I think we need to sleep for a while between two iterations, or this thread may use too much cpu resource.

Reply:
It try to take elements from a blocking queue and then block until timeout, so won't cost too much CPU